src/axom/inlet/Writer.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ *******************************************************************************
+ * \file Writer.hpp
+ *
+ * \brief This file contains the abstract base class definition of Writer.

```

Comment:
I like this abstraction.
I can imagine a use-case where one would want to output each top-level table in a separate file.